wire-ios-data-model/Tests/Source/Model/Observer/ConversationObserverTests.swift

```diff
@@ -644,37 +644,47 @@ final class ConversationObserverTests: NotificationDispatcherTestBase {
         // given
         let conversation = ZMConversation.insertNewObject(in: self.uiMOC)
         conversation.conversationType = ZMConversationType.oneOnOne
+
+        let otherUser = ZMUser.insertNewObject(in: self.uiMOC)
+
         self.uiMOC.saveOrRollback()
         XCTAssert(waitForAllGroupsToBeEmpty(withTimeout: 0.5))
 
         // when
-        self.checkThatItNotifiesTheObserverOfAChange(conversation,
-                                                     modifier: { conversation, _ in
-                                                        conversation.connection = ZMConnection.insertNewObject(in: self.uiMOC)
-                                                        conversation.connection!.status = ZMConnectionStatus.pending
+        self.checkThatItNotifiesTheObserverOfAChange(
+            conversation,
+            modifier: { conversation, _ in
+                otherUser.connection = ZMConnection.insertNewObject(in: self.uiMOC)
+                otherUser.connection?.status = ZMConnectionStatus.pending
+                otherUser.oneOnOneConversation = conversation
             },
-                                                     expectedChangedFields: ["connectionStateChanged",
-                                                                             "nameChanged"],
```

Comment:
I don't know why 'nameChanged' is here or how it relates to inserting a connection, but it is no longer marked as changed.